inc/osvr/ClientKit/Imaging.h

```diff
@@ -94,29 +92,25 @@ namespace clientkit {
                                      const struct OSVR_ImagingReport *report) {
                 ImagingCallbackRegistration *self =
                     static_cast<ImagingCallbackRegistration *>(userdata);
-                ImagingReportOpenCV newReport;
+                ImagingReport newReport;
                 newReport.sensor = report->sensor;
                 newReport.buffer.reset(report->state.data,
                                        ImagingDeleter(self->m_ctx));
-                newReport.frame = cv::Mat(
```

Comment:
Touche - good point :D Probably could split this into two layers - a lifetime-management layer and then a opencv layer on top, so you can use whichever you like. Especially with Imaging at the moment,I really don't want people using the C API because you actually get a lock on an element of a shared-memory ring buffer, so failing to release that is a problem.